docs/DeveloperGuide.md

```diff
@@ -199,9 +237,28 @@ This section describes some noteworthy details on how certain features are imple
 
 ### 4. Tabs Feature
 
-- Change tabs (By Clicking)
-- Automatically change tab based on command entered
+Since our application had two primary classes `Friends` and `Events` we needed to be able to view instances of both of these classes without the GUI being cluttered with details. Thus, we decided to implement a Tab Pane with one `Friends` tab and an `Events` tab. 
+
+The following images show how the Tabs feature look when the `Friends` tab is selected and when the `Events` tabs is selected
+
+![friendsTab.png](C:\Users\aryan\Desktop\Sem 6\CS2103\tP\src\main\resources\images\friendsTab.png) ![eventsTab.png](C:\Users\aryan\Desktop\Sem 6\CS2103\tP\src\main\resources\images\eventsTab.png)
+
+#### 4.1 Automatically Change tabs (By Command)
+
+1. `CommandResult` class was modified to contain another boolean called `event` this boolean indicates whether the command which was just executed requires us to switch to the events tab or not.
```

Comment:
Description here feels sparse - can be more thorough (say, with a code sample), but not very important.